wremnants/combine_theory_helper.py

```diff
@@ -145,7 +145,7 @@ def add_resum_unc(self, scale=1):
         if self.minnlo_unc and self.minnlo_unc not in ["none", None]:
             # sigma_-1 uncertainty is covered by scetlib-dyturbo uncertainties if they are used
             helicities_to_exclude = None if self.resumUnc == "minnlo" else [-1]
-            for sample_group in self.samples:
+            for sample_group in ["signal_samples_inctau","single_v_nonsig_samples"]:
```

Comment:
What is the reason for this change?

Reply:
This is needed to apply the QCD scale variations to signal in the theory agnostic version of the fit

Reply:
For the standard analysis nothing changes in principle, can check with the CI

Reply:
For the standard analysis yes, but we also want to keep running alternative configurations. The change seems to me to make things less flexible.

Reply:
Wouldn't this add QCD scale variations to other samples aside from the W and Z? (and also change the splitting for the W and Z samples?)

Reply:
(I would suggest looking carefully to make sure the expected number/splitting of nuisances are there)

Reply:
So actually I checked this explicitly for the standard analysis, and this change keeps the behaviour unchanged, but is actually more transparent about what is done and less error-prone, so I would actually keep it like this.